playbooks/roles/apt-packages/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+# tasks file for apt-packages
+- name: Update cache
+  apt:
+    update_cache: yes
+    cache_valid_time: 3600
+  become: yes
+
+# - name: Install nvidia driver
+#   apt:
+#     name: nvidia-driver
+#     state: latest
+#   become: yes
+
+- name: Install packages
+  apt:
+    name: "{{ packages }}"
+  vars:
```

Comment:
Maybe instead of putting the variables here, they would be better of inside the "vars" folder and include them with 
```
include_vars: '{{ vars_file }}'
```
So the role can be generic and reusable. A complete example is here: https://github.com/lerwys/lnls-ans-role-epics-dev/blob/master/tasks/main.yml